src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -124,6 +142,7 @@ public IReadOnlyList<Token> Tokenize(string expressionText)
         /// <returns></returns>
         public ParseResult Parse(string expressionText, ParserOptions options = null)
```

Comment:
>Parse [](http://example.com/codeflow?start=27&length=5)

we should test
- expressionText is not null
- expressionText.Length

Reply:
There are many parse overloads - we test them at the common point below to make sure we catch all cases.